packages/renderer/src/lib/flows/components/InputFieldActionsColumn.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+import { faEdit, faTrash } from '@fortawesome/free-solid-svg-icons';
+
+import type { InputField } from '/@/lib/flows/types/input-field';
+import ListItemButtonIcon from '/@/lib/ui/ListItemButtonIcon.svelte';
+
+interface ParameterWithCallbacks extends InputField {
+  index: number;
+  onEdit: (index: number, field: InputField) => void;
+  onDelete: (index: number, field: InputField) => void;
+}
+
+interface Props {
+  object: ParameterWithCallbacks;
+}
+
+let { object }: Props = $props();
+
+function handleEdit(): void {
```

Comment:
suggestion(non-blocking): pass responsability to build these edit and delete methods to the parent

I wrote the change in the following PR https://github.com/MarsKubeX/kortex/pull/2 if you want to take a look